### PR TITLE
Move DNX test packages into DNX.

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: src/dnx.common/tpa.cpp [hunk 1]
```diff
@@ -18,7 +18,9 @@ const std::vector<const dnx::char_t*> CreateTpaBase(bool native_images)
             _X("Microsoft.Dnx.Host.CoreClr.ni.dll"),
             _X("Microsoft.Dnx.Loader.ni.dll"),
             _X("Microsoft.Extensions.PlatformAbstractions.ni.dll"),
+            _X("Microsoft.Extensions.PlatformAbstractions.Dnx.ni.dll"),
```
Comment (davidfowl) [inline src/dnx.common/tpa.cpp]: ?

Comment (pranavkm) [inline src/dnx.common/tpa.cpp]: I built DNX and it updated this file.

Comment (davidfowl) [inline src/dnx.common/tpa.cpp]: Weird that System.Buffers is in there.

**Comment by victorhurdugaci** [comment]
Why is this change?

**Comment by pranavkm** [comment]
@victorhurdugaci this moves the DNX specific test infrastructure into DNX. This could reduce cyclc dependencies between Testing and Logging \ Common \ PlatformAbstractions.

**Comment by victorhurdugaci** [comment]
Niice! :shipit:

**Comment by muratg** [comment]
Cool! :shipit:

We're going to stop running these once RC2 is out right?

**Comment by pranavkm** [comment]
Yup. I'm going to make aspnet.xunit depend on DNX on the CI so we'll continue to get new builds of it until we hit RC2 but it won't be part of Universe-Coherence.

**Pull request merged**